docs/Makefile

```diff
@@ -12,12 +12,14 @@ build: realclean all
 pdf: manual.pdf
 html: manual.html
 
-md/manual.md: md/head.md md/intro.md md/installing.md md/ant-task.md md/example.md md/lex-specs.md \
-              md/encodings.md md/performance.md md/porting-and-parsers.md md/end.md
+md/manual.md: md/head.md md/intro.md md/installing.md \
+              md/maven-plugin.md md/ant-task.md md/example.md md/lex-specs.md \
+              md/encodings.md md/performance.md md/porting-and-parsers.md\
+              md/end.md
 	cat $^ > $@
 
 manual.html: manual.mdx
-	pandoc -S --css manual.css --filter pandoc-citeproc -f markdown $< -o $@
+	pandoc --css manual.css --filter pandoc-citeproc -f markdown+smart $< -o $@
```

Comment:
The most recent pandoc install I'm getting with macports doesn't seem to support the `+smart` target. Is it essential? What does it do?

Reply:
Just saw in the man page that this is the same as `-S` or `--smart`. Would it be Ok to leave in the command line option or does that create a conflict with newer versions?